zboss/include/osif/zb_vendor.h

```diff
@@ -244,6 +279,8 @@
 
 /* ZBOSS stack options non-configurable via Kconfig. */
 
+/* Remove upper layers when building MAC tests. */
+
```

Comment:
This looks interesting, but it is correct. The `unifdef` tool removed the `#ifndef ZB_MAC_TESTING_MODE`. I think we may keep it as-is for now :slightly_smiling_face: